src/API/Site/Controllers/Google/AccountController.php

```diff
@@ -25,6 +25,17 @@ class AccountController extends BaseController {
 	 */
 	protected $connection;
 
+	/**
+	 * Mapping between the client page name and its path.
+	 *
+	 * @var string
+	 */
+	private const NEXT_PATH_MAPPING = [
+		'setup-mc'  => '/google/setup-mc',
```

Comment:
Consider adding a comment that this first value is used as a default and changing the order of keys/values may affect things below e.g. https://github.com/woocommerce/google-listings-and-ads/pull/1069/files#diff-699dc48ef5ba31467d95145db88b190b3466fc6e935d85117262bbc5cdd72c0bL117

Reply:
Added by https://github.com/woocommerce/google-listings-and-ads/pull/1069/commits/a4fbf6fc8531bc1439214d1fdf05ff69d82c298e.